internal/usecase/gateway/google.go

```diff
@@ -0,0 +1,9 @@
+package gateway
+
+import (
+	"github.com/reearth/reearth-backend/pkg/file"
+)
+
+type Google interface {
+	FetchCsv(token string, fileId string, sheetName string) (*file.File, error)
+}
```

Comment:
This interface could use some improvement. In the future I would like to integrate it into the existing DataSource interface, but for now this is fine.